app/assets/javascripts/game.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function(){
+  $('.join').on('click', function(){
+    var gameId = //write jquery to get id - hidden field?
+    var currentUserId = //how to get this?
```

Comment:
Expected an identifier and instead saw 'var'.<br>Missing semicolon.<br>'currentUserId' is not defined.